src/components/dashboard/purchase-card.svelte

```diff
@@ -3,36 +3,33 @@
   import Button from 'ui/button.svelte';
   import UnclickableChip from 'ui/unclickable-chip.svelte';
   import parseColor from '@/utils/optimal-color';
+  import { API_HOST } from '@/constants/env.js';
+  import StockChangeStatuses from '@/constants/backend/stock-change-statuses.js';
 
   export let purchase;
 
   let dispatch = createEventDispatcher();
-  let editing = true;
-  let ready = false;
+  let editing = purchase.status === StockChangeStatuses.PENDING;
 
-  const setReady = () => {
-    ready = true;
-    editing = false;
-  };
   const copy = () => {
     if (!navigator.clipboard) alert('Browser does not support copying');
-    else navigator.clipboard.writeText(purchase.clientEmail);
+    else navigator.clipboard.writeText(purchase.account.email);
   };
 </script>
 
 <li>
   <div class="product">
     <div class="image" style={'background:' + parseColor(purchase.variety.color)}>
-      <img src={purchase.variety.image} alt="" />
+      <img src={API_HOST + purchase.variety.images[0]} alt="" />
     </div>
     <div class="product-info">
       <div class="title">
-        {purchase.name}
-        <span class="type">{purchase.type}</span>
+        {purchase.product.name}
+        <span class="type">{purchase.product.type || ''}</span>
       </div>
-      <time>{purchase.date}</time>
+      <time>{new Date(purchase.time).toLocaleString()}</time>
```

Comment:
I wrote an earlier comment about the date format, will start a conversation to allow to resolve.